hare/broker_test.go

```diff
@@ -708,12 +620,74 @@ func TestBroker_Flow(t *testing.T) {
 	b.Register(context.Background(), instanceID3)
 	b.Register(context.Background(), instanceID4)
 	b.Unregister(context.Background(), instanceID2)
-	r.Equal(instanceID0, b.minDeleted)
+	r.Nil(b.outbox[instanceID2.Uint32()])
 
 	// check still receiving msgs on ch1
 	b.HandleMessage(context.Background(), "", mustEncode(t, m.Message))
 	<-ch1
 
 	b.Unregister(context.Background(), instanceID1)
-	r.Equal(instanceID2, b.minDeleted)
+	r.Nil(b.outbox[instanceID1.Uint32()])
+}
+
+// Checks that the broker correctly enforces the limit on concurrently registered processes.
+func TestBroker_Limit1(t *testing.T) {
```

Comment:
can you put a more descriptive name other than Limit1? what's the diff from Limit2

Reply:
I've removed the limit in broker